android/hello_sdl_android/src/main/java/com/sdl/hellosdlandroid/MainActivity.java

```diff
@@ -40,4 +157,8 @@ public boolean onOptionsItemSelected(MenuItem item) {
 		}
 		return super.onOptionsItemSelected(item);
 	}
+
+	public static enum STREAM_ENUM{
```

Comment:
Can the `STREAM_ENUM` enum be defined in a separate class?

Reply:
@santhanamk this change is related more to demonstrate proposal implementation, rather than to a functional one. If it is required, I can separate this code, but, I believe, this should not go to the final release version and should be deleted. Though, it remains here right now in order to help everybody to acknowledge. Should I remove it now?